modules/ROOT/pages/fd-connection-edit.adoc

```diff
@@ -0,0 +1,31 @@
+= Edit a Connection
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+You can edit a connection that is being used by or is available to be used by a specific type of connector, changing any of the information that you have specified in it.
+
+To edit a connection, you must log into Anypoint Platform with the user ID that was used to create the connection. You cannot edit a connection that was created under a user ID that you are not logged in with.
```

Comment:
log into > log in to

Reply:
For example, I would consider reducing all the previous text to something like "If you are logged in (to the Anypoint Platform) with the same ID that created a connection, you can edit that connection, whether it is in use or not."